src/ape_test/__init__.py

```diff
@@ -57,6 +57,8 @@ class Config(PluginConfig):
     Set to ``False`` to keep providers connected at the end of the test run.
     """
 
+    hd_path: str = "m/44'/60'/0'"
```

Comment:
default path is `m/44'/60'/0'/0/{}`
https://github.com/ethereum/eth-account/blob/master/eth_account/hdaccount/__init__.py#L12

Reply:
Yes, same as this one: https://github.com/ApeWorX/ape/blob/main/src/ape/utils/testing.py#L40
It should be driven from 1 spot, as a constant, like we do for the mnemonic and number of accounts (See my other comment!)

Reply:
this value was there before me that is why i kept it, now will just change it